src/components/modals/VoteHistoryModal.vue

```diff
@@ -99,14 +99,15 @@ export default {
 }
 
 h3 {
-  margin: 0 40px;
+  margin: 0 90px;
   svg {
     vertical-align: middle;
   }
 }
 
 table {
   border-spacing: 10px 0;
+  padding: 10px 0px 0px 0px;
```

Comment:
Why not just add a bottom margin to the `h3` rule for this?

Reply:
ok